### DISCUSSION TITLE
Q: Is it hard to convert this to display the kWh?

### DISCUSSION THREAD

**Discussion by ju1ced** [ulic75/power-flow-card#30] [Q&A]
Hi,

Question, would it be hard to convert this card to be similar to the one in the energy dashboard (kWh)?
I know that one can be displayed in lovelace, but i want to use different sensors in my card than in the energy dashboard,
that way it's more realtime then the energy dashboard one.

regards
Joost

**Comment by ulic75** [comment]
It probably wouldn't be terribly hard.  But it's not the purpose of this card and it's probably not a project that I'd take on myself.

**Comment by blackbird0589** [comment]
I would also be interested in converting - in my case I have got a Growatt Inverter which displays power and energy sensors. So easiest way would be to change the displayed unit in kWh. In my opinion there's no need of any other changes to convert the flow card displaying energy (almost same like HA integrated ernergy dashboard. @ulic75: Is it possible to integrate a command to change the displayed unit?

**Reply by chipsa1982** [reply]
This is exactly my point. To clarify - the integrated energy dashboard in HA is pretty useless due to the extreme lag. Your card @ulic75 is awesome since it changes in real time. I'm using it both for real time power consumption, as it is designed, but also have another copy where I input not power, but todays energy meters (kWh data). Works like a charm, the only thing that need to be tweaked is that it still displays W or kW. AFAIK, this would be the only change - add the option to display custom units. Cheers and keep up the great work!
